tests/tls_report.log

```diff
@@ -0,0 +1,182 @@
+
+
+ Service detected:       HTTP
+
+
+[1m[4m Testing protocols [m[4mvia sockets except NPN+ALPN [m
```

Comment:
Can we strip out these non-printing characters and colour codes? It doesn't matter for golden file comparison, but does make the file less human-readable. I'm sure I had a regex for finding these colour codes once, but I can't find it now.

Reply:
I intentionally didn't, because when they mismatch, the output comes out coloured too. :)

For human readable output, just `cat tls_report.log`.

But I can try to do that on both sides, so the diff still passes, if that's the preference...

Reply:
Ah-ha, found it:
`escape_sequence_regex = re.compile(r"\033\[(\d+(?:;\d+)*)m")`

Reply:
Ah cool, I assumed the diff tool would apply its own colouring rather than retaining these colour codes! Happy to keep it for that feature in that case.